qemu/tests/boot_order_check.py

```diff
@@ -34,6 +73,7 @@ def run(test, params, env):
             process.system("ifconfig %s down" % nic.ifname)
 
     vm.resume()
+    time.sleep(5)
```

Comment:
Why sleep here?

Reply:
It takes more time before we can get complete information through "query-pci" with q35 machine type than pc machine type.

Reply:
The problem described above cannot be reproduced, so I remove the sleep() here.